Makefile

```diff
@@ -30,18 +30,19 @@ format:
 .PHONY: $(BINDIR)/test integration integration.ads integration.xds integration.rest integration.ads.tls
 
 $(BINDIR)/test:
-	@go build -race -o $@ pkg/test/main/main.go
+	echo "Building test linux binary"
+	@CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=vendor -a -tags netgo -ldflags '-w -extldflags "-static"' -o $@ pkg/test/main/main.go
```

Comment:
can you enable the race detector with `-race`?

Reply:
`-mod=vendor` is breaking the build since we should be using modules.

Reply:
> can you enable the race detector with `-race`?

I'll try and get that back in. Turning CGO off disables the `-race` flag but I can enable it again.